doc/src/guides/contributing/documentation-style-guide.rst

```diff
@@ -417,3 +417,16 @@ Avoid unwelcoming or judgement-based phrases like “That is wrong.” Instead u
 friendly and inclusive language like “A common mistake is...”
 
 Avoid extraneous phrases like, “we just have to do one more thing.”
+
+SymPy Logos
+^^^^^^^^^^^
```

Comment:
This should go on the page about building the documentation (build-docs.rst).

Reply:
Very good, I moved "SymPy Logos" section from _Documentation Style Guide_ to _Build the Documentation_ > Build the Docs, after Local Installation